Automattic/pocket-casts-android
# Display Plus Interstitial story

## Description

This adds the Plus Interstitial story UI.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_3956

## Testing Instructions

1. Sign in with a free account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the Plus Interstitial story and verify it.

## Screenshots or Screencast 

### Preview

| Regular | Small |
| - | - |
| ![Screenshot_20241018-112323](https://github.com/user-attachments/assets/c1767abf-ec86-48ca-82dd-33a101e334c0) | ![smol](https://github.com/user-attachments/assets/4b1782bd-aad3-4bb0-8729-25a222b59434) |

### Animation

https://github.com/user-attachments/assets/3dd7cb32-6f2c-4217-a00f-55047d6e9518

## Checklist
 
#### I have tested any UI changes...
